.eslintrc.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   globals: {
     $: true,
     __: true,
+    n__: true,
```

Comment:
@priley86 Thanks, this looks good.
I have one concern though -- this won't work with Gaprindashvili
@himdel Can you confirm?

I was thinking that we could add the `n` implementation after G is released, just to keep things easier right now with our `gaprindashvili` branch.

Once G is released, we could apply the `n` changes all over the app.

Reply:
@himdel @AparnaKarve can you please discuss it and give me the suggested solution for now? I would like to get past this one :)

Reply:
we can revisit this in a future PR. Since `gaprindashvili` does not currently support, I am reverting this change.

Reply:
@priley86 Thanks!

Reply:
`n__` has been supported since 2015 in our JS.

Why would it not work in gaprindasvhili?

Reply:
Where does the thing about `n__` not working in gaprindashvili come from? We have had it for quite a while actually.

Reply:
@AparnaKarve - i will add the `n__` syntax back in a separate PR later today and let you confirm it once more...

Reply:
`n__` has never been used before in our ui-classic JS sources yet, hence wanted to be doubly sure that it was OK for gaprindashvili.

@priley86  apologies for these extra i18n iterations.

With the release impending for Gaprindashvili, just being extra cautious that a wrong commit does not get in 

Thanks everyone!